src/status_ctrl.js

```diff
@@ -156,7 +157,13 @@ export class StatusPluginCtrl extends MetricsPanelCtrl {
 			s.url = target.url;
 			s.display = true;
 			s.displayType = target.displayType;
-			s.valueDisplayRegex = target.valueDisplayRegex;
+			target.valueDisplayRegexValidated = this.validateRegex(target.valueDisplayRegex);
+			if(target.valueDisplayRegexValidated) {
+				s.valueDisplayRegex = target.valueDisplayRegex;
+			}
+			else {
+				s.valueDisplayRegex = "";
+			}
```

Comment:
you could do all this section in one line in the view without an extra variable. just use `ng-show`